documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -143,7 +143,7 @@ This sector is our boot-sequence starting point. As we will see, this sector con
 
 After control is received from the BIOS at memory address `0x7c00`, [.filename]#boot0# starts executing. It is the first piece of code under FreeBSD control. The task of [.filename]#boot0# is quite simple: scan the partition table and let the user choose which partition to boot from. The Partition Table is a special, standard data structure embedded in the MBR (hence embedded in [.filename]#boot0#) describing the four standard PC "partitions". [.filename]#boot0# resides in the filesystem as [.filename]#/boot/boot0#. It is a small 512-byte file, and it is exactly what FreeBSD's installation procedure wrote to the hard disk's MBR if you chose the "bootmanager" option at installation time. Indeed, [.filename]#boot0#_is_ the MBR.
 
-As mentioned previously, the `INT 0x19` instruction causes the `INT 0x19` handler to load an MBR ([.filename]#boot0#) into memory at address `0x7c00`. The source file for [.filename]#boot0# can be found in [.filename]#sys/boot/i386/boot0/boot0.S# - which is an awesome piece of code written by Robert Nordier.
+As mentioned previously, the `INT 0x19` instruction causes the `INT 0x19` handler to load an MBR ([.filename]#boot0#) into memory at address `0x7c00`. The source file for [.filename]#boot0# can be found in [.filename]#stand/i386/boot0/boot0.S# - which is an awesome piece of code written by Robert Nordier.
```

Comment:
This reads funny. We're calling the BIOS to read the MBR and this is a rather indirect way of saying so.

Reply:
Maybe an expression like this is a better solution for now? :

**As mentioned previously, the called bios interrupt causes the `INT 0x19` handler to load an MBR** ... ?
or
**As mentioned previously, the called BIOS interrupt `INT 0x19` causes the handler to load an MBR** ... ?